mapstruct/src/main/java/com/baeldung/expression/dto/LicenseDto.java

```diff
@@ -3,13 +3,6 @@
 import java.time.LocalDateTime;
 import java.util.UUID;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-
-@AllArgsConstructor
-@Data
-@Builder
 public class LicenseDto {
 
     private UUID id;
```

Comment:
The id field is actually unused and unmapped - can we update the LicenseMapper to map this field, just to have a complete example? And add one more unit test to show the mapping. We do not need to make any changes to the article, since this change is unrelated to the topic.

Reply:
sure, usually id are populated by the database when the License entity is being used persisted, hence I didn't explicitly add any mapping. But I can dd it if it will make more sense

Reply:
done

Reply:
just saw PR is already merged, let me raise another PR